ocean/hurricane/USDEQU60at15cr5rr1WD_veg/synthetic_sandy/config_init.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<config case="init">
+	<add_link source="../../build_mesh/culled_mesh/culled_mesh.nc" dest="mesh.nc"/>
+	<add_link source="../../build_mesh/culled_mesh/culled_graph.info" dest="graph.info"/>
+	<add_link source_path="script_configuration_dir" source="scripts/create_pointstats_file.py" dest="create_pointstats_file.py"/>
+	<add_link source_path="script_configuration_dir" source="scripts/vegetation_tif_to_mpas.py" dest="vegetation_tif_to_mpas.py"/>
+	<add_link source_path="script_test_dir" source="vegetation_tif_to_mpas.config" dest="vegetation_tif_to_mpas.config"/>
+	<add_link source_path="script_configuration_dir" source="sandy_stations/NOAA-COOPS_stations.txt" dest="NOAA-COOPS_stations.txt"/>
+	<add_link source_path="script_configuration_dir" source="sandy_stations/USGS_stations.txt" dest="USGS_stations.txt"/>
+	<add_link source_path="script_configuration_dir" source="scripts/spinup_time_varying_forcing.py" dest="spinup_time_varying_forcing.py"/>
+	<add_link source_path="script_configuration_dir" source="hurricane_wind_pressure/main.py" dest="create_time_varying_forcing.py"/>
+	<add_link source_path="script_configuration_dir" source="synthetic_hurricanes/SANDY_017771.json" dest="SANDY_017771.json"/>
+	<add_link source_path="script_configuration_dir" source="synthetic_hurricanes/SANDY_017771_hurricane_inputs.txt" dest="hurricane_inputs.txt"/>
+	<add_link source_path="initial_condition_database" source="Chesapeake_and_Delaware_Bay_LambMap_DietrichN.tif" dest="manning.tif"/>
+	<add_link source_path="initial_condition_database" source="DelawareSM.tif" dest="vegetation_mask.tif"/>
+	<add_link source_path="initial_condition_database" source="Delaware.Size.2000-08.tif" dest="vegetation_size.tif"/>
+	<add_link source_path="initial_condition_database" source="Delaware.Height.2000-08.tif" dest="vegetaton_height.tif"/>
+	<add_link source_path="initial_condition_database" source="Delaware.Density.2000-08.tif" dest="vegetation_density.tif"/>
```

Comment:
@sbrus89 and @caozd999, these all need to be put on the LCRC server (Anvil) and there needs to be commands for downloading them if they're missing from the local database.  See: https://github.com/MPAS-Dev/compass/blob/4cee6c61ab7cb33eddab419ffbcdc99f593d0650/ocean/global_ocean/config_files/config_initial_state.xml#L7-L25

Reply:
I went ahead and added the files to Anvil.

```suggestion
 <get_file dest_path="initial_condition_database" file_name="Chesapeake_and_Delaware_Bay_LambMap_DietrichN.tif"> 
 	<mirror protocol="wget" url="https://web.lcrc.anl.gov/public/e3sm/mpas_standalonedata/mpas-ocean/initial_condition_database/"/> 
 </get_file>

 <get_file dest_path="initial_condition_database" file_name="DelawareSM.tif"> 
 	<mirror protocol="wget" url="https://web.lcrc.anl.gov/public/e3sm/mpas_standalonedata/mpas-ocean/initial_condition_database/"/> 
 </get_file>

 <get_file dest_path="initial_condition_database" file_name="Delaware.Size.2000-08.tif"> 
 	<mirror protocol="wget" url="https://web.lcrc.anl.gov/public/e3sm/mpas_standalonedata/mpas-ocean/initial_condition_database/"/> 
 </get_file>

 <get_file dest_path="initial_condition_database" file_name="Delaware.Height.2000-08.tif"> 
 	<mirror protocol="wget" url="https://web.lcrc.anl.gov/public/e3sm/mpas_standalonedata/mpas-ocean/initial_condition_database/"/> 
 </get_file>

 <get_file dest_path="initial_condition_database" file_name="Delaware.Density.2000-08.tif"> 
 	<mirror protocol="wget" url="https://web.lcrc.anl.gov/public/e3sm/mpas_standalonedata/mpas-ocean/initial_condition_database/"/> 
 </get_file>

 <get_file dest_path="initial_condition_database" file_name="Chesapeake_and_Delaware_Bay_LambMap_DietrichN.tif"> 
 	<mirror protocol="wget" url="https://web.lcrc.anl.gov/public/e3sm/mpas_standalonedata/mpas-ocean/initial_condition_database/"/> 
 </get_file>

	<add_link source_path="initial_condition_database" source="Chesapeake_and_Delaware_Bay_LambMap_DietrichN.tif" dest="manning.tif"/>
	<add_link source_path="initial_condition_database" source="DelawareSM.tif" dest="vegetation_mask.tif"/>
	<add_link source_path="initial_condition_database" source="Delaware.Size.2000-08.tif" dest="vegetation_size.tif"/>
	<add_link source_path="initial_condition_database" source="Delaware.Height.2000-08.tif" dest="vegetation_height.tif"/>
	<add_link source_path="initial_condition_database" source="Delaware.Density.2000-08.tif" dest="vegetation_density.tif"/>
```

Reply:
Good point, @xylar. Thank you for putting the files on Anvil.